### PR TITLE
#2 Init models

### PR DESCRIPTION
**Pull request by thinhbd-1256**

### CODE REVIEW

**Comment by thinhbd-1256** [comment]
ready

**Comment by huongnguyenmta** [comment]
commented
Teamwork - members trong team review pull cho nhau nhé -_-

**Comment by thinhbd-1256** [comment]
ready

**Comment by huongnguyenmta** [comment]
conflicted

**Comment by thinhbd-1256** [comment]
ready

File: app/models/category.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+class Category < ApplicationRecord
+  has_many :event_categories, foreign_key: "category_id"
```
Comment (huongnguyenmta) [inline app/models/category.rb]: dùng `symbol`

**Comment by huongnguyenmta** [comment]
commented

**Comment by thinhbd-1256** [comment]
ready

**Pull request merged**